model/model.wl

```diff
@@ -371,11 +357,18 @@ ParametricNDSolveValue[{
 	{r0natural, importtime}
 ];
 
+endTime[ifun_]:=Part[ifun["Domain"],1,-1];
+
 
 
+setDistancing[state_,scenario_] := Module[{},
+  distancing[t_] := stateDistancing[state, scenario["id"], t];
+];
```

Comment:
This seems odd to me. Why call this function instead of just writing that line out in the code?

Reply:
This is probably extraneous now -- will remove